src/main/webapp/i18n/de/complaintResponse.json

```diff
@@ -21,7 +21,7 @@
         },
         "moreFeedbackResponse": {
             "title": "Antwort auf die Anfrage",
-            "created": "Deine Antwort auf die weitere Feedbackanfrage wurde eingereicht",
+            "created": "Deine Antwort auf die weitere Feedbackanfrage ist eingereicht",
```

Comment:
Maybe "ist eingereicht worden" as it says "created"?

Reply:
I'll wanted to avoid passive, but I change it back to "wurde eingereicht"